spec/rails_helper.rb

```diff
@@ -67,4 +67,9 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+
+  config.append_after(:each) do
+    DatabaseCleaner.clean
```

Comment:
https://avdi.codes/configuring-database_cleaner-with-rails-rspec-capybara-and-selenium/

en fait il semble que le comportement par defaut de rspec c'est de wrapper chaque test run dans une transaction de db, ce qui isole des autres specs. Mais ce defaut est overridé dans rdv-solidarites (`use_transactional_fixtures = false`). 

Il semble que dans ce cas ce soit effectivement a nous de cleaner la db a la main entre chaque test donc plutot 👍 pour cette introduction qui devrait nettement aider a empecher ce genre de flaky specs

par contre ca va probablement ralentir la test suite car avant on ne committait pas dans la db donc ca devait probablement aller plus vite. si c'est un souci il faudrait faire un wrapping transactionnel condiitonnel comme dans le blog post